14/umbraco-commerce/tutorials/getting-started-with-commerce.md

```diff
@@ -0,0 +1,183 @@
+
+    ![Create Store](images/create-store.png)
+4. Click **Create**.
+5. Configure the **General**, **Notification**, **Order**, **Product**, and **Gift Card Settings** in the **Details** tab.
+6. Select the **User Roles** and **Users** that must have access to manage the Store in the **Permissions** tab.
+
+    ![Store Permissions](images/store-permissions.png)
+7. Click **Save**.
+
+### Setting up a Location
+
+1. Navigate to the **Settings** section.
+2. Go to **Locations** under the Store.
+3. Click **Create Location**.
+4. Enter the **Name** for the Location. For example: *Denmark*
+5. Provide the necessary address details.
+
+    ![Create Location](images/create-location.png)
+6. Click **Save**.
+
+### Setting up Order Status
+
+When you first set up Umbraco Commerce, it comes with a set of predefined order statuses to help manage the order lifecycle. These statuses usually include: New, Completed, Cancelled, and Error. These statuses can be customized depending on your specific business requirements.
+
+To create an order status:
+
+1. Navigate to the **Settings** section.
+2. Go to **Order Statuses** under the Store.
+3. Click **Create Order Status**.
+4. Enter a **Name** for the order status. For Example: *Processing*
+5. Select a **Color** for the order status.
+
+    ![Create Order Status](images/create-order-status.png)
+6. Click **Save**.
+
+### Setting up Payment Methods
+
+By default, Umbraco Commerce comes with the Invoicing and Zero Value Payment Providers. These built-in payment providers are useful for getting started. Umbraco Commerce also supports the integration of different third-party payment gateways. For more information, see the [Umbraco Commerce Payment Providers Documentation](../../../commerce-add-ons/payment-providers/README.md).
+
+To create a payment method:
+
+1. Navigate to the **Settings** section.
+2. Go to **Payment Methods** under the Store.
+3. Click **Create Payment Method**.
+4. Select a payment provider from the list. For Example: *Zero Value*.
+5. Enter a **Name** for the payment method. For example: *Zero Payment*.
+6. Configure the payment method as per your requirements.
+
+    ![Create Payment Method](images/create-payment-methods.png)
+7. Click **Save**.
+
+### Setting up Shipping Methods
+
+By default, Umbraco Commerce comes with the basic Pickup option. For more information on the integration for different providers, see the [Umbraco Commerce Shipping providers Documentation](../../../commerce-add-ons/shipping-providers/README.md).
+
+To create a shipping method:
+
+1. Navigate to the **Settings** section.
+2. Go to **Shipping Methods** under the Store.
+3. Click **Create Shipping Method**.
+4. Choose the shipping provider from the list. For Example: *DHL*.
+5. Enter a **Name** for the shipping method. For example: *DHL*.
+6. Configure the shipping method as per your requirements.
+
+    ![Create Shipping Method](images/create-shipping-method.png)
+7. Click **Save**.
+
+### Setting up a Country
```

Comment:
What does it mean to set up a country?
Is this to indicate which country products are sold to?
Would it make sense to add some more info to this part?

Also, how is country different from location?